docs/data/data-grid/row-updates/row-updates.md

```diff
@@ -39,6 +39,20 @@ In addition, the area in which `onRowsScrollEnd` is called can be changed using
 
 {{"demo": "InfiniteLoadingGrid.js", "bg": "inline", "disableAd": true}}
 
+## Lazy loading [<span class="plan-pro"></span>](https://mui.com/store/items/mui-x-pro/)
+
+To allow the grid to lazy load data, set `rowsLoadingMode="server"`.
+Then the `rowCount` needs to be set to the number of available rows on the server
+In addition, you need to handle the `onFetchRows` callback to fetch the rows for the corresponding index.
+Finally, you need to use the `apiRef.current.insertRows()` to tell the DataGrid where to insert the newly fetched rows.
```

Comment:
The demo is using `unstable_replaceRows`. Which one is the right method? BTW, if we say to use a method it shouldn't have "unstable". IHMO, "replace" is better because it can be reused for other things too.

Reply:
My bad, I forgot to update it after changing the name of the method. I was thinking whether it should be unstable or not but since the feature is that's why I added the `unstable_` prefix.
The method itself is stable so I can remove the prefix and people can use it without the lazy loading.